test/wpt/README.md

```diff
@@ -138,9 +135,9 @@ loads:
 * Status file (for example, `test/wpt/status/url.json` for `url`)
 * The WPT harness
 
-Then, for each test, it creates a vm with the globals and mocks,
+Then, for each test, it creates a child process with the globals and mocks,
```

Comment:
Aren't these worker threads?

Reply:
Good catch. I started the refactor with child processes in mind and then switched to worker threads :D